docker/twelf-app/twelf.r

```diff
@@ -288,6 +294,12 @@ resource 'styl' (rAboutText) {
   }
 };
 
+resource 'styl' (rBalloonText) {
```

Comment:
I don't think this is used.

Reply:
The reason I believe it is is the passage "The value 6 is specified here when the help message for this component is stored as styled text--that is, in both 'TEXT' and 'styl' resources." from [here](https://dev.os9.ca/techpubs/mac/MoreToolbox/MoreToolbox-199.html#HEADING199-0), but let me double-check what happens if I actually style the text...

Reply:
Ah you are right. How about using value 7 and putting the text in a `'STR '` resource?

Reply:
The style resource is used for font size information, although I wasn't able to make the color values do anything.
![image](https://github.com/jcreedcmu/twelf-mac/assets/1500822/8194d734-5cd6-4582-8764-264c32a6ad85)

Reply:
The bold/italic/shadow bits are respected: ![image](https://github.com/jcreedcmu/twelf-mac/assets/1500822/bf307db8-d541-47ef-b2ca-4cef6ad85922)

Reply:
I guess we could `STR `, since this particular string has fewer than 255 characters. I didn't love having that limitation looming. Do you prefer `STR ` just because it's more parsimonious, or for some other reason I'm not considering?

Reply:
Something may be wrong with this version of Rez's implementation of `STR ` resources. 
![image](https://github.com/jcreedcmu/twelf-mac/assets/1500822/09423d16-7611-4462-891e-c62d3b531ffb)

Reply:
I may be wrong that `STR ` tops out at 255 characters

Reply:
`Multiverse.r` has
```
type 'STR '
{
    pstring;
};
```
but I am suspicious that this is incorrect.

My current preference is to land this PR as-is with `TEXT`/`styl` and we can maybe explore fixing Retro68 upstream.

Reply:
I prefer `'STR '` since it seems wrong to override the default styling, even if we are using the same defaults.

Reply:
That is a fine reason, then. I'm on board.